engine/render/src/render/display_profiles.cpp

```diff
@@ -158,7 +158,7 @@ namespace dmRender
         float width_f = (float) width;
         float height_f = (float) height;
         float match_area = width_f * height_f;
-        float match_ratio = width_f / height_f;
+        float match_ratio = height_f > 0.0f ? (width_f / height_f) : 0.0f;
```

Comment:
When minimizing on windows both width and height becomes 0 and match_ratio becomes NaN.